src/components/EvComponents/SponsorImage.jsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from "@mui/system";
+import React from "react";
+
+const SponsorImage = (props) => {
+    return (
+        <Box sx={{
+            textAlign: "center",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
```

Comment:
give the box width and height of 150 px and make the image inside it have width 100% and height auto
This is to make sure each logo takes the same space
make sure it stays centered on small screens

Reply:
solved